src/Packagist/WebBundle/Entity/Version.php

```diff
@@ -265,6 +270,9 @@ public function toArray(array $versionData, bool $serializeForApi = false)
         if ($serializeForApi && $this->getSupport()) {
             $data['support'] = $this->getSupport();
         }
+        if ($serializeForApi && $this->getFunding()) {
```

Comment:
`$serializeForApi` should be removed here. We want to export the funding metadata in the composer repository too. Otherwise, `composer fund` would not have the funding data for packages installed from packagist (we already have this issue for `support`, but missing funding info will be a bigger issue as it makes the whole funding feature broken)

Reply:
Thank you, missed that!